demos/browser/app/meeting/meeting.ts

```diff
@@ -1128,7 +1128,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
     // TODO: enforce roster names
     new TimeoutScheduler(200).start(() => {
       const rosterName = this.roster[tileState.boundAttendeeId]
-        ? this.roster[tileState.boundAttendeeId].name
+        ? `${this.roster[tileState.boundAttendeeId].name}${tileState.boundExternalUserId ? ` [${tileState.boundExternalUserId}]` : ``}`
```

Comment:
Why does the external user ID need to be visible in the roster? Won't that cause the roster width to be very large?

Reply:
We decided in a standup to have the external id in the video tile as trial for few days.